pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -231,28 +215,54 @@ type ReconcileIntrusionDetection struct {
 	licenseAPIReady *utils.ReadyFlag
 	dpiAPIReady     *utils.ReadyFlag
 	tierWatchReady  *utils.ReadyFlag
-	usePSP          bool
+	multiTenant     bool
```

Comment:
Why are we removing usePSP ?

Reply:
PodSecurityPolicies have been deprecated since k8s 1.21 and removed since v1.25, and we now have supported their replacement (PodSecurityStandards) for several releases, so I removed it rather than refactor it for multi-tenancy.